compiler/src/dotty/tools/backend/jvm/DottyBackendInterface.scala

```diff
@@ -527,10 +527,17 @@ class DottyBackendInterface(outputDirectory: AbstractFile, val superCallsMap: Ma
 
   def assocsFromApply(tree: Tree) = {
     tree match {
+      case Block(_, expr) => assocsFromApply(expr)
```

Comment:
This produces compile-errors when I run `dotty-compiler-bootstrapped/publishLocal`

```scala
[error] /Users/ollie/dev/dotty/compiler/src/dotty/tools/backend/jvm/DottyBackendInterface.scala:530: recursive method assocsFromApply needs result type
[error]       case Block(_, expr) => assocsFromApply(expr)
[error]                              ^
```

Should I be running a different command?